### DISCUSSION TITLE
Problem with login in vercel platform

### DISCUSSION THREAD

**Discussion by qendrimkryeziu0** [vercel/community#5350] [Account Issue] [answered]
### Question

Hello Team Vercel, 

I have a problem with the login on the platform, please fix it, my email is: qendrimkryeziu18@outlook.com

Thanks,
Qendrim
<img width="1792" alt="Screenshot 2024-01-03 at 10 06 29" src="https://github.com/vercel/community/assets/148969950/fc337c2c-d1e5-4f2c-99d2-877dffcea470">

**Comment by swarnava** [comment, marked as answer]
Hi,

Please email us at registration@vercel.com from your registered email. Thank you!

**Reply by antergone** [reply]
My account was blocked for no reason. I contacted the customer service email based on the documentation, but there was no reply for 14 days.
